test/fixtures/basic/pages/posts/index.vue

```diff
@@ -17,6 +17,17 @@ import LangSwitcher from '../../components/LangSwitcher'
 export default {
   components: {
     LangSwitcher
+  },
+  async asyncData () {
+    const params = await new Promise(resolve => {
```

Comment:
I'm not really sure if there is a point in returning static object through an extra promise. I guess you might be trying to simulate more real-world case but I don't think it can really change anything in practice as `asyncData` is an async function so it will run asynchronously anyway.
This applies to all modified tests.